src/main/java/ognl/ObjectPropertyAccessor.java

```diff
@@ -31,6 +31,20 @@
  */
 public class ObjectPropertyAccessor implements PropertyAccessor {
 
+    /**
+     * Ignore detecting and invoking read method when get property value.
+     */
+    private final boolean ignoreReadMethod;
+
+
+    public ObjectPropertyAccessor() {
+        this(false);
+    }
+
+    public ObjectPropertyAccessor(boolean ignoreReadMethod) {
```

Comment:
It would be good to document this parameter here as well. Right now it's easy to find how to use it, yet users can be confused as this parameter is just passed through to `OgnlRuntime` and the real logic happens there.

Also I wonder if this is a good approach, maybe it should be a Context scoped option, eg.:

```java
OgnlContext context = Ognl.createDefaultContext(null, new ExcludedObjectMemberAccess(true));
context.putIfAbsent("ignore_read_method", Boolean.TRUE);

...

private static boolean shouldGetReadMethod(OgnlContext context) {
  Object value = context.get("ignore_read_method");
  return value instanceof Boolean ? (Boolean) value : true;
}    


if (m == null && shouldGetReadMethod(context))
  m = getReadMethod((target == null) ? null : target.getClass(), propertyName, null);
```

`getMethodValue` is getting to have too many boolean flags

Reply:
ok